.github/test-spec.yml

```diff
@@ -125,3 +125,19 @@
   - "drivers/wifi/**/*"
   - "subsys/net/**/*"
   - "subsys/settings/**/*"
+
+"CI-cloud-test":
+  - "include/zephyr/dfu/**/*"
+  - "include/zephyr/net/**/*"
+  - "include/zephyr/posix/**/*"
+  - "include/zephyr/settings/**/*"
+  - "drivers/led/**/*"
+  - "drivers/net/**/*"
```

Comment:
drivers/net seems logical to include based on the name, but we don't use any of those specific drivers